### PR TITLE
Registry doc: updating Kafka output and few other cosmetics

### PR DESCRIPTION
**Pull request by nachocano**
Fixes https://github.com/knative/eventing/issues/929

## Proposed Changes

- This is a follow up on the other registry PR, as I updated the Kafka code, it outputs another source value.
- Cosmetic changes to the rest of the doc.

### CODE REVIEW

**Comment by nachocano** [comment]
/assign @samodell 
Sam this is a follow up PR. It seems more changes than it actually is. It should be an easy approve.
I updated the kafka output, and few other cosmetics in the text. Sockpuppet I think will take care of the formatting once this lands.

File: docs/eventing/event-registry.md [hunk 1]
```diff
@@ -69,214 +64,199 @@ spec:
...
+links might help to get you started. 
 
-1. [Installing Knative](./install/README.md) in case you haven't already done
-   so.
-1. [Getting started with eventing](./eventing) in case you haven't read it.
-1. [Knative code samples](./samples/) is a useful resource to better understand
-   some of the Event Sources (remember to point them to a Broker if you want
-   automatic registration of EventTypes in the registry).
+1. [Installing Knative](./install/README.md) in case you haven't already done so.
+1. [Getting started with eventing](./eventing) in case you haven't read it.
```
Comment (samodell) [inline docs/eventing/event-registry.md]: "Getting started with eventing" is a broken link.

File: docs/eventing/event-registry.md [hunk 2]
```diff
@@ -69,214 +64,199 @@ spec:
...
+
+    When you `kubectl apply` this yaml, the KafkaSource `kafka-source-sample` will be instantiated,
+    and two EventTypes will be added to the registry (as there are two topics). You can see that in the registry 
+    example output from the previous sections.  
+        
 ## What's next
 
-We suggest the reader to experiment in her own cluster with the different Event
-Sources listed above. The following links might help to get you started.
+We suggest the reader to experiment in her own cluster with different Event Sources. The following
```
Comment (samodell) [inline docs/eventing/event-registry.md]:
```suggestion
To get started, install Knative Eventing if you haven't yet, and try experimenting with different Event Sources in your Knative cluster.
```

File: docs/eventing/event-registry.md [hunk 3]
```diff
@@ -69,214 +64,199 @@ spec:
...
+    When you `kubectl apply` this yaml, the KafkaSource `kafka-source-sample` will be instantiated,
+    and two EventTypes will be added to the registry (as there are two topics). You can see that in the registry 
+    example output from the previous sections.  
+        
 ## What's next
 
-We suggest the reader to experiment in her own cluster with the different Event
-Sources listed above. The following links might help to get you started.
+We suggest the reader to experiment in her own cluster with different Event Sources. The following 
+links might help to get you started.
```
Comment (samodell) [inline docs/eventing/event-registry.md]: Line 257 -- delete.

File: docs/eventing/event-registry.md [hunk 4]
```diff
@@ -69,214 +64,199 @@ spec:
...
-  EventTypes `source` field, which is equal to the CloudEvent source attribute.
-
-  When you `kubectl apply` this yaml, the KafkaSource `kafka-source-sample` will
-  be instantiated, and two EventTypes will be added to the registry (as there
-  are two topics). You can see that in the registry example output from the
-  previous sections.  
-
-
+    As Manual Registration might be tedious and error-prone, we also support automatic registration of EventTypes. 
+    Herein, the creation of the EventTypes is done upon instantiation of an Event Source.
```
Comment (samodell) [inline docs/eventing/event-registry.md]: What does "upon instantiation" of an Event Source mean? Is that just when an Event Source is installed (kubectle apply)?

Comment (samodell) [inline docs/eventing/event-registry.md]:
```suggestion
    The creation of the EventTypes is done upon instantiation of an Event Source. 
```

Comment (nachocano) [reply docs/eventing/event-registry.md]: yes, that is correct!

Comment (nachocano) [reply docs/eventing/event-registry.md]: Done, thanks!

File: docs/eventing/event-registry.md (hunk 2 above)
Comment (nachocano) [reply docs/eventing/event-registry.md]: Done thanks!

File: docs/eventing/event-registry.md (hunk 3 above)
Comment (nachocano) [reply docs/eventing/event-registry.md]: Done, thanks!

File: docs/eventing/event-registry.md (hunk 1 above)
Comment (nachocano) [reply docs/eventing/event-registry.md]: Changed it to ./README.md, thanks!

**Comment by nachocano** [comment]
@samodell done with the changes. I need to focus on some important code fixes. 
IMO this can go in, and then we can keep on iterating. What do you think?

**Comment by samodell** [comment]
Agreed! Thanks, @nachocano !

/lgtm
/approve

**Pull request merged**